File: SmartBikeRental/ClientApp/src/app/core/models/autogenerated/dtos/deviceDto.ts


export class DeviceDto
{
    deviceName: string;
    bikeRentalLED: boolean;
    bikeRentalLatitude: number;
    bikeRentalLongitude: number;
    bikeRentalTaken: boolean;

    constructor(deviceName: string = null,bikeRentalLED: boolean = null,bikeRentalLatitude: number = null,bikeRentalLongitude: number = null,bikeRentalTaken: boolean = null,)
    {
    
        this.deviceName = deviceName;
        this.bikeRentalLED = bikeRentalLED;
        this.bikeRentalLatitude = bikeRentalLatitude;
        this.bikeRentalLongitude = bikeRentalLongitude;
        this.bikeRentalTaken = bikeRentalTaken;
    }
}

File: SmartBikeRental/Models/DTOs/DeviceDto.cs
namespace SmartBikeRental.Models.DTOs
{
    public class DeviceDto
    {
        public string DeviceName { get; set; } = String.Empty;
        // False = locked, True = unlocked
        public bool BikeRentalLED { get; set; }
        public double BikeRentalLatitude { get; set; }
        public double BikeRentalLongitude { get; set; }
        public bool BikeRentalTaken { get; set; }
    }
}
